aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud authentication with Azure Active Directory B2C
+author: camsoper
+description: Shows how to set up Azure Active Directory B2C with ASP.NET Core
+keywords: ASP.NET Core,authentication,AAD,B2C,Azure Active Directory,Active Directory
```

Comment:
I understand the presence/absence of keywords metadata goes by individual repo SEO rules, but what was the source of the guidance that keywords aren't used for this doc set?

Reply:
keywords are not supposed to be used anymore. Source: https://microsoft.sharepoint.com/teams/CSI_SEO/Shared%20Documents/Content%20SEO%20guidance/seo-for-writers-cheat-sheet.pdf

Reply:
Thanks @mairaw ... The templates just say 'go by whatever your doc set SEO rules dictate.' I have been removing them as I go (and not adding them to new topics).

Reply:
I've been doing the same